backend/routes/creators.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+const jwt = require('jwt-simple');
+
+const{Creator}=require('../data/models');
+const router = express.Router();
+router.use(express.json());
+
+router.get('/:id', (req, res, next)=>{
+    Creator.findOne({where: {id: req.params.id} })
+    .then((creator) => res.send(creator))
+    .catch(next);
+});
+
+router.delete('/:id', (req, res, next)=>{
```

Comment:
do we have a use case where we're deleting a Creator?

Reply:
the user might want to delete their account

Reply:
alright :+1:. we just didn't have a use case for it when we were discussing before.